test/basic/src/test/java/io/narayana/lra/arquillian/FailingParticipantCallsIT.java

```diff
@@ -63,24 +64,22 @@ public void testFailingAfterLRA() throws LRACallbackException {
 
             lra = URI.create(response.readEntity(String.class));
             lrasToAfterFinish.add(lra);
-        } finally {
-            if (response != null) {
-                response.close();
-            }
-        }
+            response.close();
```

Comment:
I am not sure but maybe response needs to be nulled after https://github.com/jbosstm/lra/blob/75ba42d53acdc6e94a74dbc4ef0982c0775602f8/test/basic/src/test/java/io/narayana/lra/arquillian/FailingParticipantCallsIT.java#L67 because otherwise it seems it might reissue it if there was a problem on https://github.com/jbosstm/lra/blob/75ba42d53acdc6e94a74dbc4ef0982c0775602f8/test/basic/src/test/java/io/narayana/lra/arquillian/FailingParticipantCallsIT.java#L69 ?

Reply:
close () is idempotent so I think this change is ok @tomjenkinson (see https://jakarta.ee/specifications/restful-ws/3.0/apidocs/jakarta/ws/rs/core/response#close())

Reply:
Thank you @marcosgopen